company_website/view_helpers.py

```diff
@@ -24,3 +28,15 @@ def add_meta_tags_to_page_context(page_name: str, context_data: Dict) -> Dict:
     context_data["meta_description"] = page_seo.meta_description
     context_data["keywords"] = page_seo.keywords
     return context_data
+
+
+def generate_subresource_integrity_sha384(content_link: str) -> str:
+    try:
+        response = requests.get(content_link, timeout=5)
```

Comment:
Timeout might be a constant out here.

Reply:
Also 5 seconds is quite big timeout I think. It might be better if it would be like 0.5 second or something. If there would be a waiting up to 5 seconds every request page rendering might take like more than 10 seconds. I need to test it on test server to see if it is working well

Reply:
I just tested it and I cannot see any delay caused by this request but timeout still should be reduced like to 0.5 or something. Besides I'm wondering when there will be empty SRI would this widget work or not? I'm thinking now about it and I don't like either generating SRI nor leaving this without any hash :p

Reply:
Constant done. I don't know if you can put flot in there. But it's only 10 sec when both pages are not responding. I've reduced  it to 3 sec.

Reply:
>  Besides I'm wondering when there will be empty SRI would this widget work or not? I'm thinking now about it and I don't like either generating SRI nor leaving this without any hash :p

Why would it be empty? I've just tested empty byte string, and there is a hash for it :D but obviously widget would not work, because it need's that content.